app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -31,6 +31,9 @@ child :content_view_environments => :content_view_environments do
   node :label do |cve|
     cve.label
   end
+  node :id do |cve|
```

Comment:
How to test this with hammer command?

Reply:
`hammer -d host info --id xxx` and look at the response json for content_view_environments.

Reply:
In the hammer debug output, look at the response json. Not the regular Hammer output.

Reply:
I think this name `id` conflicts with the one on line 1. Renamed it to `cve_id` then it showed up in the hammer debug output.

Reply:
Finally I see this :+1: